sunpy/data/data_manager/downloader.py

```diff
@@ -42,6 +40,8 @@ class ParfiveDownloader(DownloaderBase):
     """
 
     def download(self, url, path):
+        from sunpy.util.parfive_helpers import Downloader
```

Comment:
This is basically a hack. We need to be able to import `sunpy.data` for our tests, but the user API requires parfive. I am open to better suggestions.